index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
> I haven't found anything about ext and dir in the readme.

Thats because the Node API doesn't have I/O logic for files, like build/task runners

> They can be calculated from from and to that are there. 

From user perspective ```ctx.from => ctx.dir && ctx.ext``` is more convenient

```js
module.exports = (ctx) => ({
   parser: ctx.ext === '.sss' ? : 'sugarss' : false
   plugins: {
     'postcss-import': { root: ctx.dir }
   }
})
```

> I don't want to pass options to the config loader, because this goes against the point of having this config in the first place. Anything that is sent to postcssLoadConfig can become a local constant in the config file.

**gulpfile.js**
```js
const css = () => {
   return src('src/*.css')
     .pipe(postcss())
     .pipe(dest(./dest))
}

const cssm = () => {
    return src('src/components/*.css')
     .pipe(postcss({ modules: true }))
     .pipe(dest(./dest))
}

task('default', series(css, cssm))
```
**postcss.config.js** 
```js
module.exports = (ctx) => ({
   parser: ctx.ext === '.sss' ? : 'sugarss' : false
   plugins: {
     'postcss-import': { root: ctx.dir }
     'postcss-modules': ctx.modules ? {} : false
   }
})
```